include/seqan3/core/detail/all_view.hpp

```diff
@@ -0,0 +1,167 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Enrico Seiler <enrico.seiler AT fu-berlin.de>
+ * \brief Provides seqan3::detail::all.
+ */
+
+//!\cond
```

Comment:
why?

Reply:
We said that we don't want to document it for now, because lots of work.

We could more or less just link to
https://eel.is/c++draft/range.zip
and
https://en.cppreference.com/w/cpp/ranges/zip_view

Reply:
Ah, this was all_view. docs are follow up :o

Reply:
Hm. Can you at least add some documentation in the details section of the class itself? As to why we need to implement it. I fear that a documentation follow up will not be done soon..